internal/controller/numaflowcontrollerrollout_controller_test.go

```diff
@@ -19,44 +19,46 @@ package controller
 import (
 	"context"
 	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	apiv1 "github.com/numaproj/numaplane/pkg/apis/numaplane/v1alpha1"
 )
 
 var _ = Describe("NumaflowControllerRollout Controller", Ordered, func() {
 	const (
-		namespace    = "default"
-		resourceName = "numaflow-controller"
+		namespace = "default"
+		duration  = 20 * time.Second
+		interval  = 1 * time.Second
 	)
+	Context("When creating a NumaflowControllerRollout resource", func() {
+		ctx := context.Background()
 
-	ctx := context.Background()
-
-	Context("When reconciling a resource", func() {
-		typeNamespacedName := types.NamespacedName{
-			Name:      resourceName,
-			Namespace: namespace,
+		resourceLookupKey := types.NamespacedName{
+			Name:      "numaflow-controller",
+			Namespace: "default",
 		}
 
-		numaflowControllerRollout := apiv1.NumaflowControllerRollout{
+		numaflowControllerResource := apiv1.NumaflowControllerRollout{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      resourceName,
-				Namespace: namespace,
+				Name:      "numaflow-controller",
+				Namespace: "default",
```

Comment:
Why not reuse the namespace const defined above since it is the same?

Reply:
yaa, sorry I missed it